lib/nerves_ssh/options.ex

```diff
@@ -17,7 +17,8 @@ defmodule NervesSSH.Options do
 
   alias Nerves.Runtime.KV
 
-  @default_system_dir "/etc/ssh"
+  @default_host_system_dir "/etc/ssh"
+  @default_device_system_dir "/data/nerves_ssh"
```

Comment:
The host/device naming is tripping me up. Just to make sure that I understand, is the `host_system_dir` for ssh host keys and the `device_system_dir` for ssh client keys?

Reply:
I take that back. There won't be any client keys since this is an ssh server. So all of the keys are host keys? Now I'm less sure that I know the difference. My next guess is that `host_system_dir` is the system directory for host keys (e.g., the system`/etc` directory). And `device_system_dir` is the non-system directory that this code uses for storing host keys.

Reply:
Ya, maybe not the best names

The typical Linux practice is storing in `/etc/ssh` and is what Erlang SSH defaults to. The context here is we really only see this on "host" systems running/testing this locally, or if someone has created a system with `/etc/ssh` in use

The "device" system dir is where we'd expect it to be on device.

Reply:
`/etc/ssh` could be used on device if host keys are stored in the the `rootfs_overlay`, and that might be something people want to do. It would be an easy way of restoring the current `nerves_ssh` semantics of having a hard coded server (aka host) key.

I think that we should make clear in this context that "host" is used in the ssh sense where it means server, and not overload it in the host/device sense.

I like these names and I don't feel like they need to be user configurable.

```elixir
  @system_ssh_host_key_dir "/etc/ssh"
  @nerves_ssh_data_dir "/data/nerves_ssh"
```